src/Parsers/ASTSelectQuery.h

```diff
@@ -32,7 +32,8 @@ class ASTSelectQuery : public IAST
         LIMIT_BY,
         LIMIT_OFFSET,
         LIMIT_LENGTH,
-        SETTINGS
+        SETTINGS,
+        INTERPOLATE
```

Comment:
I think INTERPOLATE should be a part or ORDER_BY ast.

Reply:
ORDER BY is a list of elements - we don't have an ast for it, the same for INTERPOLATE